Signal/src/environment/Environment.m

```diff
@@ -215,6 +215,24 @@ + (void)presentConversationForRecipientId:(NSString *)recipientId
     });
 }
 
++ (void)presentConversationForThreadId:(NSString *)threadId
+{
+    OWSAssert(threadId.length > 0);
+
+    __block TSThread *thread;
+    [[TSStorageManager sharedManager].dbReadWriteConnection
+        readWriteWithBlock:^(YapDatabaseReadWriteTransaction *_Nonnull transaction) {
+            thread = [TSThread fetchObjectWithUniqueID:threadId];
```

Comment:
What?  Shouldn't this be `[fetchObjectWithUniqueID: transaction:]` or ... why not just use `[fetchObjectWithUniqueID:]` without the wrapping transaction?

Reply:
thank you!